core/unit_test/TestAtomicOperations.hpp

```diff
@@ -536,7 +536,7 @@ bool DivAtomicTest( T i0, T i1 )
 
   bool passed = true;
 
-  if ( (resSerial-res)*(resSerial-res) > 1e-10 ) {
+  if ( abs(resSerial-res) > 1e-5 ) {
```

Comment:
At some point in the future (not in this PR), we should fix the error bounds to depend on T.